adds feature request template

pablo-gar opened the pull request:
Adds an issue template for users providing a feature request.

pablo-gar commented on .github/ISSUE_TEMPLATE/feature-request.md:
```diff
@@ -0,0 +1,25 @@
+
+## Description
+
+Provide a detailed description of the request
```
@atolopko-czi I'm assuming using quotes here will take care of the automated addition of the label "user request"

atolopko-czi commented on .github/ISSUE_TEMPLATE/feature-request.md:
```diff
@@ -0,0 +1,25 @@
+
+## Description
+
+Provide a detailed description of the request.
```
Since this section is asking for _detailed_ description, we might not need the "ideal behavior" section below. Alternately, the "ideal behavior" section could be changed to something like "What other alternatives have you considered?"

pablo-gar commented:
Thanks for your comments @pablo-gar I addressed them all. I inspired the template from the `single-cell-data-portal` template. I added one section based on the other templates.

I think this one is a slight improvement, unless you have any strong concerns about consistency then I'd prefer to keep it.
